services/frontend-service/src/ui/components/ReleaseDialog/ReleaseDialog.scss

```diff
@@ -86,18 +86,21 @@
           top: -10px;
           left: -10px;
           height: 42px;
-        }
-        .env-card-label {
-          svg.env-card-env-lock-icon {
-            width: 24px;
-            height: 24px;
-            margin: unset;
-          }
-
-          .env-card-env-locks:nth-child(2) {
-            margin-left: 55px;
+          .env-card-label {
             display: flex;
-            flex-direction: row;
+
+            svg.env-card-env-lock-icon {
+              width: 24px;
```

Comment:
Don't leave values as these ones in the code. You can see these standards in the knowledge base ([here](https://revolution.dev/app/-JqFGExX46gs9mH7vxR5/WORKSPACE_DOCUMENT/-Mn5HLuHv2wjoq0QKPax/)) Add these variables to the .scss variables in the `_variables.scss` ([here](https://github.com/freiheit-com/kuberpult/blob/main/services/frontend-service/src/assets/_variables.scss))